Update OntoMetadataShape.ttl - suggestions from WGTS

SArndt-TIB opened the pull request.

SArndt-TIB committed:
Update OntoMetadataShape.ttl - title - langString constraint

SArndt-TIB commented on src/shapes/OntoMetadataShape.ttl:
```diff
@@ -45,9 +45,9 @@ ontom:OntologyMetadataShape a sh:NodeShape ;
         sh:datatype rdf:langString ;
-        sh:message "The ontology title is not an rdf:langString, i.e. it does not carry a language tag."@en ;
+        sh:message "The ontology title is not an rdf:langString, i.e. it does not carry a language tag. We recommend to add one."@en ;
         sh:description "The ontology title must be an rdf:langString, i.e. it does not carry a language tag."@en ;
```
```suggestion
        sh:description "The ontology title must be an rdf:langString, i.e. it does not carry a language tag. We recommend to add one."@en ;
```

StroemPhi replied:
We should then also change 
`sh:description "The ontology title must be an rdf:langString, i.e. it does not carry a language tag."@en`
to 
`sh:description "This shape checks whether the ontology title literal value is an rdf:langString. It will produce an info message if the title does not carry a language tag."@en`

SArndt-TIB replied:
I updated the suggestion accordingly.

SArndt-TIB committed:
Update OntoMetadataShape.ttl - prefix/acronym > minCount xone

StroemPhi commented on src/shapes/OntoMetadataShape.ttl:
```diff
@@ -45,9 +45,9 @@ ontom:OntologyMetadataShape a sh:NodeShape ;
     sh:property [             
         sh:path [sh:alternativePath (rdfs:label skos:prefLabel dc:title dcterms:title schema:name) ] ; 
-        sh:severity sh:Violation ;
+        sh:severity sh:Info ;
```
Not sure, but think starting rather lax might be better and then slowly tightening the screws in future iterations.

SArndt-TIB replied:
Okay, then I guess sh:Info is the friendliest way to communicate our suggestions to ontology engineers - friendlier than a Warning.

StroemPhi replied:
Yes, agreed.

SArndt-TIB committed:
Update src/shapes/OntoMetadataShape.ttl

Update sh:description of property shape for ontology title

SArndt-TIB committed:
Update OntoMetadataShape.ttl

update property shape for ontology license

SArndt-TIB commented on src/shapes/OntoMetadataShape.ttl:
```diff
@@ -45,78 +45,61 @@ ontom:OntologyMetadataShape a sh:NodeShape ;
-        sh:message "The ontology title is not an rdf:langString, i.e. it does not carry a language tag."@en ;
-        sh:description "The ontology title must be an rdf:langString, i.e. it does not carry a language tag."@en ;
+        sh:message "The ontology title is not an rdf:langString, i.e. it does not carry a language tag. We recommend to add one."@en ;
+        sh:description "This shape checks whether the ontology title literal value is an rdf:langString. It will produce an info message if the title does not carry a language tag."@en ;
```
```suggestion
        sh:description "This shape checks whether the ontology title literal value is an rdf:langString. It will produce an info message if the title does not carry a language tg."@en ;
```

SArndt-TIB committed:
Update OntoMetadataShape.ttl

update property shapes for versionIRI

SArndt-TIB commented on src/shapes/OntoMetadataShape.ttl:
```diff
@@ -45,78 +45,61 @@ ontom:OntologyMetadataShape a sh:NodeShape ;
-        sh:description "The ontology must declare its preferred namespace prefix."@en ;
-    ] ;
+### G-Doc https://docs.google.com/document/d/1FRmCQ7eD6PcqSQgPbqCGjki3t39ROoYozsehCPX9E0Q/edit#bookmark=id.1iqmwa13y7u
+    sh:xone (
```
```suggestion
    sh:or (
```

SArndt-TIB committed:
Update OntoMetadataShape.ttl

update creator shape

SArndt-TIB committed:
Update OntoMetadataShape.ttl

remove examples class shape

SArndt-TIB committed:
Update src/shapes/OntoMetadataShape.ttl

update pref namespace prefix/ acronym

SArndt-TIB committed:
Update OntoMetadataShape.ttl

set modification date to sh:severity Info and update sh:message and sh:description

SArndt-TIB committed:
Update OntoMetadataShape.ttl

update contributor shape

SArndt-TIB committed:
Update OntoMetadataShape.ttl

lower severities

SArndt-TIB committed:
Update OntoMetadataShape.ttl

add examples and instructions to creator, license, title, prefix shapes

SArndt-TIB committed:
Update OntoMetadataShape.ttl

update version info shapes: versionIRI, creation date, modification date

SArndt-TIB committed:
Update OntoMetadataShape.ttl

update messages for required metadata

SArndt-TIB committed:
Update OntoMetadataShape.ttl

update recommended metadata shapes

SArndt-TIB committed:
Update OntoMetadataShape.ttl

update optional metadata shapes

SArndt-TIB committed:
Update OntoMetadataShape.ttl

update creator and contributor example statements

SArndt-TIB committed:
#7 Updates from WGTS discussion on 2023-11-22

SArndt-TIB committed:
#7 updates for version info, media type and file format

SArndt-TIB committed:
#7 updates from WGTS discussion

SArndt-TIB committed:
#7 clean up repo
